tekton/integration_pipeline_test.go

```diff
@@ -56,11 +60,26 @@ var _ = Describe("Integration pipeline", func() {
 					"test.appstudio.openshift.io/optional": "false",
 				},
 			},
-			Spec: v1alpha1.IntegrationTestScenarioSpec{
+			Spec: v1beta1.IntegrationTestScenarioSpec{
 				Application: "application-sample",
-				Bundle:      "quay.io/kpavic/test-bundle:component-pipeline-pass",
-				Pipeline:    "component-pipeline-pass",
-				Environment: v1alpha1.TestEnvironment{
+				ResolverRef: v1beta1.ResolverRef{
+					Resolver: "git",
+					Params: []v1beta1.ResolverParameter{
+						{
+							Name:  "url",
+							Value: "https://github.com/redhat-appstudio/integration-examples.git",
```

Comment:
Optional: What do you think about a new ITS using Git resolver which points to `integration_pipelinerun_fail.yaml`? to test the negative scenario in unit tests

Reply:
Since the integration Tekton pipelines are not actually executed in unit tests (only their CRs are created), I don't think that it would make a difference which pipeline definition it points to, apart from having some visibility into which pipelines exist in our examples repo.

Reply:
I think we can skip adding this for now.